src/components/secrets/Secret.tsx

```diff
@@ -290,24 +295,59 @@ export const SecretForm: React.FC<SecretFormProps> = function (props) {
     }
 
     async function handleSubmit(e) {
+        const secretNameRegex = new RegExp(PATTERNS.CONFIGMAP_AND_SECRET_NAME);
+
         if (secretMode) {
             toast.warn(<ToastBody title="View-only access" subtitle="You won't be able to make any changes" />)
             return
         }
         if (!configName.value) {
-            setName({ value: "", error: 'Field is manadatory' })
+            setName({ value: "", error: 'This is a required field' })
             return
         }
-
-        if (!/^[-.a-zA-Z0-9]+$/.test(configName.value)) {
-            setName({ value: configName.value, error: 'Name must be of format /^[-.a-zA-Z0-9]+$/' })
+        if (configName.value.length > 253) {
+            setName({ value: configName.value, error: 'More than 253 characters are not allowed' })
```

Comment:
check with vikram, if he is prefixing the name then u cannot have 253 characters

Reply:
he is not prefixing anything in the name